### PR TITLE
added missing work from my branch that disappeared

### PR DESCRIPTION
**Pull request by TamaraJoniec**
added missing work from my branch that disappeared in the main

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,7 +11,8 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css"
     integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N" crossorigin="anonymous">
     <link rel="stylesheet" href="./assets/style.css">
-
+    <link rel="stylesheet"
```
Comment (omovo) [inline index.html:14]: Checked

File: assets/style.css [hunk 2]
```diff
@@ -51,10 +51,17 @@ h1{
...
 .card-body {
     background: rgb(240,248,255);
-    background: linear-gradient(0deg, rgba(240,248,255,1) 47%, rgba(148,187,233,1) 100%);
+    background: linear-gradient(0deg, #4353ab 27%, #0C1940 100%);
 }
 
+.subheading {
+    font-family: 'Martel', serif;
+    color: #F2F2F2;
+    font-size: 40px;
```
Comment (omovo) [inline assets/style.css:60]: Checked

**Review by omovo — APPROVED** [review]
Checked

File: assets/style.css [hunk 3]
```diff
@@ -51,10 +51,17 @@ h1{
 /* Tamara's work right-side */
 .card-body {
     background: rgb(240,248,255);
-    background: linear-gradient(0deg, rgba(240,248,255,1) 47%, rgba(148,187,233,1) 100%);
+    background: linear-gradient(0deg, #4353ab 27%, #0C1940 100%);
 }
 
+.subheading {
```
Comment (TamaraJoniec) [inline assets/style.css:57]: Checked

**Review by TamaraJoniec — COMMENTED** [review]
Checked

**Pull request merged**